### ISSUE TITLE
[Idea] Change radar and alienssense from beacons back to blips for enemy players

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/gameplay#43]
Related Problem issues: #3

Before the beacons we had blips, small circles at the edges of the screen that only displayed the ruffly direction of enemies and distance by fading.
Compared to the beacons whit their absolute positions this system is much more atmospheric and leaves room for interpretation.
It also prevents as in #3 that Snipers can kill dretches instantly cause the beacons act like wallhacks. 

**Disclaimer**
>Ideas without a related problem issue and lacking explanation or depth are closed after a certain time.

**Comment by slipher** [comment]
+1, I also dislike beacons for enemy players.

**Comment by bmorel** [comment]
+1

**Comment by necessarily-equal** [comment]
The commit that removed it is this one:
```
commit de14734523e6095ccb03a11e2f2455995b80ad1b
Date:   Thu Mar 5 01:58:20 2015 +0100

    Remove old human radar and alien sense.
```

reverting this commit could be surprisingly easy

**Comment by bmorel** [comment]
Indeed. It seems most of the changes are files moving around and the rocket to rmlui transition.

**Comment by Viech** [comment]
Fwiw, I think that for Aliens at least this is an atmospheric feature in the sense that it makes them appear to have superhuman reflexes/speed. It was agreed upon in #3 that the human radar in its current form is too strong, this does not mean one has to remove this feature from aliens as well.

**Comment by bmorel** [comment]
From a recent discussion on IRC, someone (ZdrytchX?) preferred the old radar system for alien radars, claiming it was easier to keep track of enemies with it.
Me, I do not have enough memories of the old on-screen blips to remember. And even remembering, I would probably need to be able to switch between them to judge anyway, I was a weak alien player before the beacons (me being now a decent player is more linked to the training, I don't think it's related to this change)

**Comment by illwieckz** [comment]
I would also like to return to blips for enemy players, at least for humans and maybe only with a radar?

I like the idea of aliens having greater sense than humans, but that should not be too strong, so for example it may look OK to me if aliens have beacons while humans have blips without radar, but aliens having blips may be enough if humans have no blip by default (like tremulous).

The beacon mechanism can be kept for bases (at least friendly ones) and telling “go to the medipad there”, and for manually placed beacons.

I see player blips as a proximity sensor, while bases are shared memory (through overmind or human technology), the later explains why beacons for enemy bases don't disappear until someone go to see if it's still there or not. Such lore element may also help to define things.

**Comment by Viech** [comment]
There appears to a level of agreement concerning the human radar but some controversy/less enthusiasm concerning the alien sense. I suggest to concentrate here (first) on the human capabilities for player tagging (to resolve #3).

Since this is an idea issue, can the replacement mechanic for (1) humans being able to tag enemy players by looking at them and (2) the human radar _item_ be fleshed out? The original issue reads as if either one of them or both should be replaced with Trem's alien sense blips but maybe this is not what was meant.

**Comment by Gireen** [comment]
blips for all enemies and beacons only for any building and teammates
tagged enemy players should also be displayed with blips

the problem from #3 also exists for aliens but with humans its worse
so i think it would be good bring blips back for both teams

the human radar then would be the mechanical version of the aliensense

**Comment by Viech** [comment]
I do not like tagging (= staring at) enemies to enable blips for them. I think the staring mechanic should be reserved for creating beacons that appear in the location you are looking at, even if this limits that mechanic to buildables (which I'd be fine with).

With the remainder, i.e. beacons for teammates, friendly buildables, and spotted enemy buildables, and blips for enemy players if you are either an alien or carry the radar, I could live.

One caveat to watch out for is that a straightforward implementation of this change removes the teamplay element from the radar. (I would be against providing blips to teammates who do not carry the radar. This does not seem intuitive.) However, maybe an ability to automatically tag alien structures could be added to the radar to preserve some of its teamplay value.

**Comment by bmorel** [comment]
Actually, I change my vote to: no, do not do that.

In theory, I don't like the beacons as we have them currently. But in practice, it's the only reliable way to see the aliens, as they blend too much.
